frontend/packages/multicluster-sdk/src/api/useFleetClusterNames.ts

```diff
@@ -0,0 +1,17 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk'
+import { UseFleetClusterNames } from '../types/fleet'
+
+const ManagedClusterListGroupVersionKind = {
+  group: 'clusterview.open-cluster-management.io',
+  version: 'v1',
+  kind: 'ManagedCluster',
+}
+export const useFleetClusterNames: UseFleetClusterNames = () => {
+  const [clusters] = useK8sWatchResource<[]>({
+    groupVersionKind: ManagedClusterListGroupVersionKind,
+    isList: true,
+  })
+
+  return clusters
```

Comment:
I think we need to return the loaded and error items from the tuple.

Do we want to extract just the names from the clusters? You've indicated that the return type is `string[]` but this looks like it will return the full `ManagedCluster` resources?

Reply:
Ack, oversight. Thank you. 

I've updated the code.